src/08-accessing.js

```diff
@@ -171,7 +189,22 @@ function getFullAddress(person) {}
       numbers: [7185550921, 7185558611],
     };
  */
-function getFlatObject(person) {}
+function getFlatObject(person) {
+  newObj = {};
+
+newObj.first = person.names.first;
+newObj.middle = person.names.middle;
```

Comment:
let move all this over one tab.  Also - think about how we could do this with a loop :)